src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1651,6 +1651,20 @@ def from_json(cls, json_rgw_spec):
         return RGWSpec(**args)
 
 
+class GrafanaServiceSpec(ServiceSpec):
+    def __init__(self, cert_fname=None, pkey_fname=None, placement=None):
+        super(GrafanaServiceSpec, self).__init__(None, placement)
+        self.cert_filename = cert_fname
```

Comment:
Are filename okay here or should we have the content of the certificates already at this point?